.vscode/settings.json

```diff
@@ -0,0 +1,6 @@
+{
+    "emmet.includeLanguages": {
+        "javascript": "javascriptreact"
+    },
+    "emmet.triggerExpansionOnTab":true
+}
```

Comment:
What is this for?

Reply:
that is done

Reply:
Hey mate, do we need a vscode/setting.json in our app?